### ISSUE TITLE
Accessibility: The state of the menu link is not conveyed to screen readers

### ISSUE THREAD

**Issue by Yalin2023** [wet-boew/cdts-sgdc#1179]
**Issue Description**:
When screen reader users select one of the link on the left menu, the selected state of the link is not conveyed to screen readers.
CDTS Section menu sample: https://cdts.service.canada.ca/app/cls/WET/gcweb/v5_0_1/cdts/samples/sectionmenu-en.html#

![image](https://github.com/wet-boew/cdts-sgdc/assets/133782445/51ff7391-9992-483f-830d-286b2a93e126)
![image](https://github.com/wet-boew/cdts-sgdc/assets/133782445/c9274556-7ead-4411-956b-aea10733d133)

**Comment by ahmad-shahid** [comment]
Hello, what is the expected behaviour when you say "selected state of the link is not conveyed"? 

Is there a recommended solution?

**Comment by shawnthompson** [comment]
It should be telling screen reader users that the link is the current page. Shown visually by the colour of the active page.

See [aria-current - Accessibility | MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current) for more information about how to make this happen.
